lib/manager/components/deployment/PeliasPanel.js

```diff
@@ -181,10 +172,14 @@ class PeliasPanel extends Component<Props, State> {
   }
 
   render () {
-    const { deployment } = this.props
-    const { peliasWebhookUrl } = this.state
+    const { deployment, project } = this.props
 
-    const optionsEnabled = peliasWebhookUrl !== ''
+    // Don't show panel is no webhook url is set
```

Comment:
Would that be better if the panel were disabled, with a note indicating how to enable it?

Reply:
I think until the Pelias instances can be deployed from within datatools, the feature should be hidden from those not interested in it (unless it were easy to set up)